PayButton/paybutton-server
# Feat/411 4 get address transactions

Testing steps:

1. run against the DB: `select * from paybutton.Transaction` (maybe also `delete from paybutton.Transaction` to clean this table)
2. open this url to sync transactions (with any bch or ecash address): http://localhost:3000/api/address/transactions/sync/ecash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7quss2vxek
3. Select again on Step 1 above and see the transactions imported there

This PR already changes in master the choice of the blockchain client from grpc to chronik for both BCH and XEC since most of chronik is already implemented and transactions had a big impact on code, so I had to remove grpc code on transactions, it no longer works.

Also I had to change some tests since the return type of some functions related to transactions changed, so I went on fixing whatever was broken by these changes, midway I realized this PR would be huge if I went on that path and I just commented out all tests that used chronik features, this will be addressed on the final #411 PR.
